.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# Description
+<!-- Explain here the changes your PR introduces and text to help us understand the context of this change. -->
+
+# Contribution Checklist:
+- [ ] **The pull request does not introduce any breaking changes**
+- [ ] **I have read the [contribution guidelines](https://github.com/oohsai/rust/blob/master/CONTRIBUTING.md).**
+- [ ] **Create an issue and link to the pull request.**
```

Comment:
I saw you opened an empty ticket with just at title and an then opened a pull request to "fix" the ticket you created a minute before.
I don't understand the benefit of opening a ticket that contains no details whatsoever https://github.com/rust-lang/rust/issues/116699 and then closing it via pr again, over just opening a pr that contains all necessary information to begin with.